config/smtp.yml.template

```diff
@@ -0,0 +1,21 @@
+default: &default
```

Comment:
I'd include some comments as a header, and a pointer to:
https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration

I'd also include the entire list of settings, a notable absent is `:openssl_verify_mode` as some people may be tripped by the use of self-signed SSL certs in their organisation.

Also, are we better of providing blank defaults, or would it be better to provide some clearly bogous details: "mail.starfleet.com", port 123456, etc. I suspect some errors may slip through the cracks with blank values but may become more evident if some crazy defaults are provided.

Reply:
✅